cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/vagrant/nitro-NITRO-2.7/build/.waf-1.7.8-77e2cdd38ea2930f982e3aabf335fdb7/waflib/Tools/c_config.py", line 460, in run_c_code\n    bld.compile()\n  File "/vagrant/nitro-NITRO-2.7/build/.waf-1.7.8-77e2cdd38ea2930f982e3aabf335fdb7/waflib/Build.py", line 198, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 21123728: c test.c -> test.c.1.o}\n[\'/usr/bin/gcc\', \'-DHAVE_INTTYPES_H=1\', \'-DHAVE_UNISTD_H=1\', \'-DHAVE_GETOPT_H=1\', \'-DHAVE_MALLOC_H=1\', \'-DHAVE_SYS_TIME_H=1\', \'-DHAVE_DLFCN_H=1\', \'-DHAVE_FCNTL_H=1\', \'../test.c\', \'-c\', \'-o\', \'test.c.1.o\']\n'
